packages/server/src/api/queueRoutes.test.ts

```diff
@@ -75,16 +51,25 @@ describe("Queue Routes", () => {
       });
     });
     it("POST new question", async () => {
-      await expectWithServer({
+      const server = getServer();
+      const request = await server.inject({
         method: "post",
-        url: "/api/v1/queues/45/questions",
+        url: "/api/v1/queues/1/questions",
         payload: {
           text: "Don't know recursion",
           questionType: "Concept",
         },
-        statusCode: 200,
-        result: MOCK_CREATE_QUESTION_RESPONSE,
       });
+      expect(request.statusCode).toEqual(201);
+      expect(request.result).toMatchObject({
+        text: "Don't know recursion",
+        taHelped: null,
+        helpedAt: null,
+        closedAt: null,
+        questionType: "Concept",
+        status: "Queued",
+      });
+      expect(QuestionModel.count({ where: { queueId: 1 } })).toEqual(1);
```

Comment:
could you duplicate this line to right before the POST? except with expected count as `0` instead.

Reply:
Good call, just added that in.